content/pages/index/onboarding/steps/claimTokens.json

```diff
@@ -0,0 +1,12 @@
+{
+  "title": "Get some test tokens",
+  "subtitle": "To publish or consume the assets in the portal you'll need some GX and OCEAN test tokens to spend",
+  "body": "Click on the buttons below to claim your test tokens.",
+  "image": "../../../../../src/images/onboarding/claim_tokens.png",
+  "gxButtonLabel": "Claim 1 GX",
+  "existingGxBalance": "You already have some GX test tokens in your wallet 🎉",
+  "gxSuccess": "Successfully claimed 1 GX",
+  "oceanButtonLabel": "Claim 100 OCEAN",
+  "existingOceanBalance": "You already have some OCEAN test tokens in your wallet 🎉",
+  "oceanSuccess": "Successfully claimed 100 OCEAN"
```

Comment:
thought(blocking): This could be formatted more elegantly with an array:
```json
"buttons": [
  {
    "name": "gxButton",
    "label": "Claim 1 GX",
    "balance": "You already have some GX test tokens in your wallet 🎉",
    "success": "Successfully claimed 1 GX"
  },
  {
    ...
  }
]
```

This also enables a "better" query in the component (see below)